<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>520</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>367</width>
      <height>241</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Circle</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>177</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Input radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="radius">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>260</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ellipse</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>72</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="b_box_input">
         <property name="maximum">
          <number>520</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>73</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="a_box_input">
         <property name="maximum">
          <number>520</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ð¡olor</string>
     </attribute>
     <widget class="QPushButton" name="fg_color_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>291</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>fg color</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="fg_prev">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="bg_prev">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bg_color_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>291</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>bg color</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="alg_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>canonical equation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>parametric equation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bresenham</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mid-point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>standart</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amount_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="draw_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>386</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="circle">
       <property name="text">
        <string>circle</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>240</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="ellipse">
       <property name="text">
        <string>ellipse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="concentric_button">
       <property name="text">
        <string>concentric circles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clean_button">
       <property name="text">
        <string>clean canvas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="step_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>310</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Input center coordinates:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="center_x">
         <property name="maximum">
          <number>520</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="center_y">
         <property name="maximum">
          <number>520</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
